hometask2/hometask2_1.js

```diff
@@ -0,0 +1,145 @@
+const express = require("express");
+const app = express();
+
+const bodyParser = require("body-parser");
+app.use(bodyParser.json() );
+app.use(bodyParser.urlencoded({
+  extended: true
+})); 
+
+
+const Joi = require("@hapi/joi");
+const validator = require("express-joi-validation").createValidator({});
+
+const schema = Joi.object({
+	login: Joi.string()
+		.alphanum()
+		.min(5)
+		.max(30)
+		.required(),
+
+	password: Joi.string()
+		.pattern(new RegExp("^[a-zA-Z0-9]{8,30}$"))
+		.required(),
+
+	age: Joi.number()
+		.integer()
+		.min(4)
+		.max(130)
+		.required(),
+});
+
+
+
+const usersMap = new Map();
+let uid = 1;
+
+//read
+app.get("/user/:id", (req, res) => {
```

Comment:
Consider renaming this file to either "index" or "app" or "server" to follow a certain naming convention.
Consider creating a separate folder "routes" at root level right next to the current file and under that folder create all the necessary folders for your routes. E.g. "routes/user/*.js" would hold the routes for your user CRUD operations.